### DISCUSSION TITLE
自定义 tooltip 的问题

### DISCUSSION THREAD

**Discussion by RJM1996** [antvis/S2#2617] [Q&A]
![image](https://github.com/antvis/S2/assets/33800707/d9ebb879-3cdb-42fb-a002-7a44bb8e3c2c)
圈选几个单元格之后，这个“3项已选择”的tooltip能够再自定义吗？比如后面加个按钮

**Comment by lijinke666** [comment]
你可以自己重写 Tooltip 的内容

https://s2.antv.antgroup.com/manual/basic/tooltip#%E8%87%AA%E5%AE%9A%E4%B9%89
